src/util/carddb.ts

```diff
@@ -147,7 +155,7 @@ export function getMostReasonable(
   ids = cards.map((card) => card.details.scryfall_id);
```

Comment:
Since we have the cards + details here, we could reverse that array rather than creating the id map and returning that. That would reduce the calls to cardFromId (at end of this method and within reasonableId called per id). Though I dunno which would be more efficient, reversing (in place) an array of "big" objects or fewer id lookups on the dictionaries.